ArraysHomework/ArraysHomework.cs

```diff
@@ -51,7 +59,7 @@ public void Arrays()
 
             // Swap the last and first value of this array.
             string[] swapMyValues = { "first", "middle", "last" };
-            
+            System.Array.Reverse(swapMyValues);
```

Comment:
What if there were more than three elements in the array, or we didn't know how many elements there were? `Reverse` will make more changes than we want.